src/Microsoft.ML.ImageAnalytics/EntryPoints/ImageAnalytics.cs

```diff
@@ -4,10 +4,10 @@
 
 using Microsoft.ML;
 using Microsoft.ML.EntryPoints;
-using Microsoft.ML.ImageAnalytics.EntryPoints;
+using Microsoft.ML.ImageAnalytics;
 
 [assembly: LoadableClass(typeof(void), typeof(ImageAnalytics), null, typeof(SignatureEntryPointModule), "ImageAnalytics")]
-namespace Microsoft.ML.ImageAnalytics.EntryPoints
+namespace Microsoft.ML.ImageAnalytics
 {
     internal static class ImageAnalytics
```

Comment:
>ImageAnalytics [](start = 26, length = 14)

Humm, is it really good to have a class named by its namespace? Once I ran into missing-reference problem because of that.